src/MSBuild.Sdk.SqlProj/Sdk/Sdk.targets

```diff
@@ -126,9 +126,15 @@
         <DacpacFile>%(_ResolvedPackageReference.PhysicalLocation)/tools/%(Identity).dacpac</DacpacFile>
         <DatabaseVariableLiteralValue>%(PackageReference.DatabaseVariableLiteralValue)</DatabaseVariableLiteralValue>
       </_ResolvedPackageReference>
+
+      <!-- Resolve all project references to their physical location -->
+      <_ResolvedProjectReference Include="%(_ResolvedProjectReferencePaths.Identity)">
+        <DacpacFile>%(_ResolvedProjectReferencePaths.Identity)</DacpacFile>
```

Comment:
Interesting, I have been struggling with how to do this with code... Inside VS extension.

Reply:
;) Took some binary logging to figure out how this works. There are some [docs](https://github.com/dotnet/msbuild/blob/master/documentation/ProjectReference-Protocol.md) on how it works if you're interested.